Makefile.pre.in

```diff
@@ -1577,9 +1582,9 @@ FROZEN_FILES_OUT = \
ude/floatobject.h Include/bytesobject.h Include/compile.h Include/pytypedefs.h Include/cpython/monitoring.h Include/cpython/abstract.h Include/cpython/setobject.h Include/intrcheck.h Include/cpython/odictobject.h Include/cpython/object.h Include/longobject.h Include/bltinmodule.h Include/internal/pycore_lock.h Include/cpython/classobject.h Include/typeslots.h Include/cpython/descrobject.h Include/pymem.h Include/pyerrors.h Include/cpython/initconfig.h Include/warnings.h Include/cpython/ceval.h Include/abstract.h Include/cpython/warnings.h Include/cpython/fileutils.h Include/complexobject.h Include/memoryobject.h Include/cpython/longobject.h Include/cpython/pyhash.h Include/import.h Include/pyhash.h Include/cpython/pylifecycle.h Include/cpython/cellobject.h Include/cpython/listobject.h Include/pyatomic.h Include/genericaliasobject.h Include/iterobject.h Include/pylifecycle.h Include/traceback.h Include/cpython/pymem.h Include/cpython/pystate.h Include/enumobject.h Include/pyport.h Include/cpython/memoryobject.h Include/cpython/code.h Include/Python.h Include/dictobject.h Include/objimpl.h Include/cpython/tracemalloc.h Include/cpython/methodobject.h Include/pymacconfig.h Include/cpython/pyctype.h Include/pythread.h Include/pystats.h Include/weakrefobject.h Include/cpython/objimpl.h Include/structseq.h Include/cpython/pyfpe.h Include/cpython/critical_section.h Include/pystrtod.h Include/cpython/bytesobject.h Include/unicodeobject.h Include/modsupport.h Include/tupleobject.h Include/cpython/import.h Include/cpython/genobject.h Include/object.h Include/lock.h Include/pymath.h Include/internal/pycore_hashtable.h Include/rangeobject.h Include/cpython/pythread.h Include/pystrcmp.h Include/codecs.h Include/boolobject.h Include/exports.h Include/osmodule.h Include/pymacro.h Include/pystate.h Include/cpython/pytime.h Include/cpython/funcobject.h Include/cpython/lock.h Include/moduleobject.h Include/fileobject.h Include/cpython/tupleobject.h Include/cpython/weakrefobject.h Include/cpython/pyatomic.h Include/cpython/unicodeobject.h Include/listobject.h Include/internal/pycore_import.h Include/monitoring.h Include/cpython/sysmodule.h Include/cpython/traceback.h Include/patchlevel.h Include/pycapsule.h Include/pythonrun.h Include/sliceobject.h Include/sysmodule.h Include/cpython/pyframe.h Include/internal/pycore_fileutils.h Include/cpython/longintrepr.h Include/cpython/pythonrun.h pyconfig.h Include/cpython/picklebufobject.h Include/cpython/compile.h Include/fileutils.h Include/marshal.h Include/cpython/floatobject.h Include/ceval.h Include/cpython/pydebug.h Include/cpython/fileobject.h Include/bytearrayobject.h Include/cpython/modsupport.h Include/descrobject.h Include/pyframe.h Include/cpython/bytearrayobject.h Include/methodobject.h Include/critical_section.h Include/cpython/complexobject.h Include/cpython/dictobject.h Include/cpython/pyatomic_gcc.h Include/cpython/pyerrors.h Include/cpython/context.h Include/refcount.h Include/setobject.h Include/pybuffer.h
```

Comment:
Shouldn't we use `$(srcdir)/Include` instead of `Include` directly? (and same remark on the ordering)